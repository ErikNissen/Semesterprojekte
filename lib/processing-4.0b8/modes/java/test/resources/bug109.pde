version https://git-lfs.github.com/spec/v1
oid sha256:a6680fdbb6e4f45c15d7cfd58ab23b75acad68cf7ea2ba2877b4c554142d9a38
size 44
