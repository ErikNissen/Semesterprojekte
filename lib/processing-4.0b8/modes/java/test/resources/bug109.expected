version https://git-lfs.github.com/spec/v1
oid sha256:46ebe27bff9522603ef33e9677a4595b71c12e491a7a4a46e406c5a40c0f68ca
size 56
